README.md

```diff
@@ -35,6 +35,24 @@ This tutorial will provide a gentle introduction to all three areas. Additionall
 * Introduction to `deepcell-tf`
 * Training a model in Google Colab
 
+## Publications
+
+Each of the following publications makes use of the DeepCell ecosystem, as well as the specific resources relevant for each paper.
```

Comment:
Eventually, we should separate publications that establish pieces of the DeepCell ecosystem vs plug ins/makes use of the ecosystem. For example, ark-analysis uses the DeepCell but the Kiosk is part of DeepCell. So, for now, I think this would be better written: "To learn more about the various systems and software that comprise DeepCell, please refer to the publications below. Relevant links are highlighted below each publication."